picard/ui/widgets/scripttextedit.py

```diff
@@ -176,15 +176,16 @@ def highlightBlock(self, text):
 
 class ScriptCompleter(QCompleter):
     def __init__(self, parent=None):
-        super().__init__(sorted(self.choices), parent)
-        self.setCompletionMode(QCompleter.CompletionMode.UnfilteredPopupCompletion)
-        self.highlighted.connect(self.set_highlighted)
+        super().__init__(self.choices, parent)
+        self.setCompletionMode(QCompleter.CompletionMode.PopupCompletion)
```

Comment:
If we change this to `PopupCompletion` we can also remove this line, as `PopupCompletion` is the default.

Reply:
Done in https://github.com/metabrainz/picard/pull/2635/commits/ee2c11dc95251861816e9c078bc89ac92cd0b3cd